GOVERNANCE.md

```diff
@@ -0,0 +1,52 @@
+# Submariner Governance
+
+## Open Governance Checklist
+
+The opengovernance.dev project [provides the following
+checklist](https://github.com/opengovernance/opengovernance.dev/blob/master/README.md#checklist) for defining open governance.
+
+Submariner's answers are provided for each point.
+
+This is not meant to define Submariner's governance, but instead to document it clearly. Authoritative sources are cited inline.
+
+### Ownership
+
+* Who owns the copyright on contributed code?
+  * All Submariner code is linted to verify the copyright notices specify "Contributors to the Submariner project", per the
```

Comment:
Should maybe link to the linting config: https://github.com/submariner-io/submariner/blob/ed473895ece6dc9be6f9d925a3cfbe441513cf17/.licenserc.json#L22

Reply:
Done in the latest.